modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/util/TaxonomyCategoryBriefUtil.java

```diff
@@ -48,6 +52,33 @@ public static TaxonomyCategoryBrief toTaxonomyCategoryBrief(
 		};
 	}
 
+	private static Scope _getScope(AssetCategory assetCategory) {
+		Group group = GroupLocalServiceUtil.fetchGroup(
+			assetCategory.getGroupId());
+
+		if (group == null) {
+			return null;
+		}
+
+		return new Scope() {
+			{
+				setExternalReferenceCode(group::getExternalReferenceCode);
+				setType(() -> _getScopeType(group));
+			}
+		};
+	}
+
+	private static Scope.Type _getScopeType(Group group) {
+		if (group.isCMS()) {
+			return Scope.Type.CMS;
+		}
+		else if (group.isDepot()) {
+			return Scope.Type.ASSET_LIBRARY;
+		}
```

Comment:
I am worried about a different type apart from CMS Depot or Site. What do you think about adding another if clause for the SITE and if not, return null?

Something like:

```java
else if (group.isSite()) {
	return Scope.Type.SITE;
}
```